CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1 @@
+[cygnus-ngsi-ld] Creation of the new cygnus-ngsi-ld bundle for receiving NGSI-LD notifications, the first sink provided is PostgreSQL
```

Comment:
Better to include issue number in CHANGES_NEXT_RELEASE when it exists, in this case:

```suggestion
[cygnus-ngsi-ld] Creation of the new cygnus-ngsi-ld bundle for receiving NGSI-LD notifications, the first sink provided is PostgreSQL (#1875)
```

Sorry not having realized before... (the additional lines and comments on this file "shadowed" this)